WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -61,10 +61,11 @@ struct Scaler {
         return CGPoint(x: CGFloat(x.timeIntervalSince(startDate)) * xScale, y: CGFloat(y - glucoseMin) * yScale)
     }
 
-    func rect(for range: WatchDatedRange) -> CGRect {
-        let a = point(range.startDate, range.minValue)
+    func rect(for range: WatchDatedRange, future: Bool = false) -> CGRect {
```

Comment:
"future" as a boolean is a little confusing. Something like "futureOnly" would be clearer to me, but I think a better route would be to have an optional start date, so it'd be:

```
func rect(for range: WatchDatedRange, overrideStartDate: Date? = nil) -> CGRect {
  let startDate = overrideStartDate ?? range.startDate
```

that achieves the same purpose, but is a more flexible API.

Reply:
actually you can collapse it further to:

```
let a = point(overrideStartDate ?? range.startDate, range.minValue)
```